src/Core/Binding/BindingConfiguration.cs

```diff
@@ -37,7 +37,7 @@ public static BindingConfiguration CreateBoundConfiguration(BoundSonarQubeProjec
             return new BindingConfiguration(project, sonarLintMode);
         }
 
-        internal /*for testing purposes*/ BindingConfiguration(BoundSonarQubeProject project, SonarLintMode mode)
+        public BindingConfiguration(BoundSonarQubeProject project, SonarLintMode mode)
```

Comment:
This was only being called from tests before - why does it need to be public now?

Reply:
It wasn't, it's also called from SolutionBindingOperation. I changed it to use the static method, and then a bunch of tests failed, since the tests pass null.